src/Components/Logo/stubs/logo.php

```diff
@@ -15,6 +15,16 @@
 
     'enabled' => true,
 
+    /*
+    |--------------------------------------------------------------------------
+    | Logo Name
+    |--------------------------------------------------------------------------
+    |
+    | This value is your logo.
```

Comment:
Can you make sure you add 3 paragraphs, where each one is 3 chars shorter than the previous one?

Reply:
How about something like:
```suggestion
    | This value determines the text that is rendered for the logo.
    | It defaults to the app name, but it can be any other text
    | value if the logo should be different to the app name.
```